homeassistant/components/broadlink/switch.py

```diff
@@ -145,6 +146,11 @@ def assumed_state(self):
         """Return true if unable to access real state of entity."""
         return True
 
+    @property
+    def available(self):
+        """Return True if entity is available."""
+        return not self.should_poll or self._is_available
```

Comment:
Why do you use a different code like other class?

Reply:
Only the switches that poll for data that can support `available`
Instead I can duplicate `available` to only the switches that supports polling